### ISSUE TITLE
On Windows 7, it uses that style for the ComboBox, which leads to height issues

### ISSUE THREAD

**Issue by saulgreenberg** [saulgreenberg/Timelapse#38]
One user reports that the ComboBox has a windows 7 look and feel, which as a side effect means that the text displayed has the bottom of it truncated (due, I think, to different heights).The snapshot of that person's display shows that only the ComboBox uses the Windows7 style; all others use the flat style. Unsure as to why that is.

**Comment by saulgreenberg** [comment]
Partially resolved, but a hack: I changed the height of various controls (text, choice, counters) to 30 (it was 25).  A test module was sent to the user to see if it now works for him. I would rather keep those to 25, as it is somewhat neater. Still, it works and doesn't consume extra screen real estate.

I did try to force the PresentationStyle to Windows10 (PresentationStyle.Luna), but kept getting an error (in spite of including the reference) when trying to Source=PresentationStyle.Lune in App.xml
